browser/src/control/jsdialog/Widget.OverflowManager.ts

```diff
@@ -58,6 +58,10 @@ class OverflowManager {
 				' req: ' +
 				requiredWidth,
 		);
+
+		// not yet known width -> do not assume it is small to prevent scrollbars
+		if (requiredWidth === 0) return true;
```

Comment:
I think this loosens the check here. This will group the buttons if the buttons have not initialized yet. But they should be grouped when there is not enough space. This real condition is omitted due to zero width.

Can we also wait some time before grouping the buttons and check the actual width?

Also, the comment here is a little confusing. We are meaning the scroll bars of the main container right? Not the scroll bar of the item here. I think we need a longer comment here if we want to keep this current state.

Thanks :)

Reply:
Since you want to get this in, let me test first, then we can hopefully merge:)